### PR TITLE
query slider data from backend

### PR DESCRIPTION
**Pull request by TolyaDeveloper**
## Description

Slider data have been hardcoded before, now, data are taken queried from backend

#### Screenshots

|         Original          |         Updated          |
| :-----------------------: | :----------------------: |
| Clip2net_220405200458 | Clip2net_220405201235 |

|         Original          |         Updated          |
| :-----------------------: | :----------------------: |
| | Clip2net_220405201342 |

### Checklist

- [x] 🔽 My branch is up-to-date with "development" branch
- [x] ✅All tests passed locally
- [x] ✨My changes working with up-to-date admin and back-end part locally, like charm
- [x] 🔗 Link pull request to issue

### CODE REVIEW

File: src/pages/home/slider/slider-home-page.style.js [hunk 1]
```diff
@@ -101,21 +109,19 @@ export const useStyles = makeStyles(() => ({
     display: 'flex',
     flexDirection: 'column',
     justifyContent: 'space-between',
+    alignItems: 'flex-start',
     '@media (max-width: 768px)': {
-      minHeight: '5rem'
-    },
-    '@media (max-width: 320px)': {
-      minHeight: '5rem'
+      minHeight: '6rem'
```
Comment (rostyslaw27) [inline src/pages/home/slider/slider-home-page.style.js:114]: I think it's better to use px cause you used px before

File: src/tests/unit/pages/home/slider/slider-home-page.spec.js [hunk 2]
```diff
@@ -1,81 +1,39 @@
...
-        {' '}
-        <SliderHomePage />{' '}
-      </Router>
-    );
-    const slider = screen.getByAltText('arrow right');
-
-    fireEvent.click(slider);
-    expect(screen.getByText(/02/)).toBeInTheDocument();
+  beforeEach(() => {
+    act(() => {
```
Comment (rostyslaw27) [inline src/tests/unit/pages/home/slider/slider-home-page.spec.js:15]: Render under the hood already uses act

**Review by rostyslaw27 — CHANGES_REQUESTED** [review]

File: src/pages/home/slider/slider-home-page.style.js (hunk 1 above)
Comment (TolyaDeveloper) [reply src/pages/home/slider/slider-home-page.style.js:114]: Agree

File: src/tests/unit/pages/home/slider/slider-home-page.spec.js (hunk 2 above)
Comment (TolyaDeveloper) [reply src/tests/unit/pages/home/slider/slider-home-page.spec.js:15]: Got, thanks

**Review by rostyslaw27 — APPROVED** [review]

**Review by maarekua — APPROVED** [review]

**Pull request merged**